vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -451,6 +451,7 @@ type VolumeCreateRequest struct {
 type NetworkResource struct {
 	Name       string                      // Name is the requested name of the network
 	ID         string                      `json:"Id"` // ID uniquely identifies a network on a single machine
+	Created    string                      // String representation of the creation timestamp
```

Comment:
Note:
- `Container.Created` is `int64`
- `ContainerJSONBase.Created` is `string`
- `Image.Created` is `int64`
- `ImageHistory.Created` is `int64`
- `ImageInspect.Created` is `string`
- `swarm/Meta.CreatedAt` (not `Create`!) is `time.Time`

https://godoc.org/github.com/docker/engine-api/types
https://godoc.org/github.com/docker/engine-api/types/swarm